docs/src/cli-reference/TensileCreateLibrary.rst

```diff
@@ -0,0 +1,133 @@
+\-\-jobs=CPUTHREADS, \-j CPUTHREADS
+    Number of parallel jobs to launch. If this options is set higher than *nproc* the number of parallel 
+    jobs will be equal to the number of cores. If the this option is set below 1 (e.g. 0 or -1), the number
+    of parallel jobs will be set to the number of cores, up to a maximum of 64. (default = -1).    
+\-\-lazy-library-loading
+    Loads Tensile libraries when needed instead of upfront.
+\-\-library-format={yaml,msgpack}
+    Select which library format to use (default: msgpack).
+\-\-library-print-debug
+    Solutions will print enqueue info when enqueueing a kernel.
+\-\-no-enumerate
+    Do not run rocm_agent_enumerator.
+\-\-no-library-print-debug
+    Solutions will not print enqueue info when enqueueing a kernel (default).
+\-\-no-merge-files
+    Store every solution and kernel in separate file.
+\-\-no-short-file-names
+    Disables short files names.
+\-\-num-merged-files=NUMMERGEDFILES
+    Number of files the kernels should be written into.
+\-\-merge-files
+    Store all solutions in single file (default).
+\-\-package-library 
+    Enable the packaging of the client library code objects into separate (deprecated).
+    architectures as sub-directories within the Tensile/library directory.
+\-\-short-file-names
+    On windows kernel names can get too long. 
+    Converts solution and kernel names to serial ids (default).
+\-\-separate-architectures
+    Separates TensileLibrary file by architecture to reduce the time to load the library file.
+    The separate-architectures option writes each architecture into a different TensileLibrary_gfxXXX.dat 
+    file.
+\-\-verbose=PRINTLEVEL, \-v PRINTLEVEL
+    Set printout verbosity level {0, 1, 2}.
+\-\-version=VERSION
+    Version string to embed into library file.
+\-\-write-master-solution-index
+    Output master solution index in csv format including number 
+    of kernels per architecture post build in csv format.
+
+Examples
+--------
+
+No options
+^^^^^^^^^^
+
+The following command will invoke *TensileCreateLibrary*
+with no options passing the Logic directory containing 
+logic files and creates a directory *tensile-output* 
+in the directory where the *TensileCreateLibrary* 
+command was invoked. The *tensile-output* directory
+will contain the artifacts.
+
+.. code-block::
+
+    TensileCreateLibrary <absolute path to>/Logic tensile-output HIP
+
+Adding TensileCreateLibrary options 
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The following example illustrates how to add options When
+invoking *TensileCreateLibrary*. In some cases, such as ``--seprate-architectures``
+no arguments are required; whereas, for ``--jobs`` an argument is required.
+
+.. code-block::
+
+    TensileCreateLibrary --separate-architectures --jobs=32 <absolute path to>Logic tensile-output HIP
+
+
+.. \-\-cmake-cxx-compiler=CMAKECXXCOMPILER
+    This doesn't appear to do much and I would like to consider removing
```

Comment:
Should we delete this from the file, mark as deprecated, or keep it and move it back up to the **Options** section?

Reply:
Yes I would like to remove it.